File: CodeForces/201-300/231/A/231A.cs
// Problem 231A: Team
// https://codeforces.com/contest/231/problem/A
// Compiled with: Mono JIT compiler version 6.12.0.199 (tarball Fri May  2 12:22:51 UTC 2025)
// Linux version: Linux kernel 6.17.0-5-generic
// Submitted on: October 26th, 2025
// Codeforces language used: C# Mono 6.8

using System;
using System.Linq;

class Program {
    static void Main() {
        int runs = int.Parse(Console.ReadLine());
        int total = 0;
        for (int i = 0; i < runs; i++) {
            int w = Console.ReadLine().Split().Select(int.Parse).ToArray().Sum();
            if (w >= 2) {
                total += 1;
            }
        }
        Console.WriteLine(total);
    }
}

File: CodeForces/201-300/231/A/231A.py
# Problem 231A: Team
# https://codeforces.com/contest/231/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: June 14th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

problems = int(input())
total_solved = 0
for _ in range(problems):
    input_string = str(input()).split()
    if sum([int(solution) for solution in input_string]) > 1:
        total_solved += 1
print(total_solved)
